### ISSUE TITLE
iOS Click overlay close not working.

### ISSUE THREAD

**Issue by danieliser** [PopupMaker/Popup-Maker#539]
https://wordpress.org/support/topic/is-it-possible-to-tap-overlay-to-close-on-mobile/

Per support ticket the following solution appears to work.

.pum-overlay { cursor: pointer; }

iOS only triggers a click event on elements with the pointer cursor or an element with a natural click event. Divs do not have a natural click event.

**Comment by JCM-Jeff** [comment]
Hi Daniel,

Maybe not elegant solution, but borrowing from the close button, I added the popmake-close class to the parent containing the overlay, and it works on my iPhone X running the most up to date iOS. It didn't have any impact on styles or other functionality that I could find.

**Comment by danieliser** [comment]
Hmm so you made the overlay itself a close button? That should also effectively make all content in the Popup a close button as well.

**Comment by JCM-Jeff** [comment]
I added this to the content using the Raw JS element in visual composer:

<script>
window.onload = function() {
  var element = document.getElementById("pum-11299");
element.classList.add("popmake-close");
};
</script>

I just tried it again and clicked on the content inside the popup, which contains text and a button link, and it didn’t close the popup. I’m not really a developer - just cobble my way around, but I was testing with Safari’s inspector, and adding that class to the parent seemed to work.

Site I was doing it on is https://smkazoo.com/ <https://smkazoo.com/> - the mobile popup is set to go off after two pages.
